week2/assignment/index.js

```diff
@@ -0,0 +1,270 @@
+import { todos as mockTodos } from "./mock.js";
+
+const input = document.querySelector(".todo-input");
+const addBtn = document.querySelector(".add-btn");
+const prioritySelect = document.querySelector(".priority-select");
+const todoList = document.querySelector(".todo-list");
+
+const todosAll = document.querySelector(".todos-all");
+const todosFinished = document.querySelector(".todos-finished");
+const todosUnFinished = document.querySelector(".todos-unfinished");
+const todosPriority = document.querySelector(".todos-priority");
+
+const deleteBtn = document.querySelector(".del-btn");
+const finishedBtn = document.querySelector(".fin-btn");
+const checkAll = document.querySelector(".check-all");
+const modal = document.querySelector(".modal");
+const closeBtn = document.querySelector(".modal-close");
+
+function createTodo(todo) {
```

Comment:
```suggestion
function createTodo({id, priority,completed,title}) {
```
구조 분해 할당하면 요소를 쉽게 사용할 수 있어용
ref: https://ko.javascript.info/destructuring-assignment

Reply:
아 맞네요. 구조분해할당 쓰는게 더 나을것 같네요. 좋은 충고 감사합니다.😊